geode-redis/src/main/java/org/apache/geode/redis/internal/netty/NettyRedisServer.java

```diff
@@ -99,7 +99,7 @@ public NettyRedisServer(Supplier<DistributionConfig> configSupplier,
       throw new IllegalArgumentException("Redis port cannot be less than 0");
     }
     this.bindAddress = getBindAddress(requestedAddress);
-    selectorGroup = createEventLoopGroup("Selector", false, 1);
+    selectorGroup = createEventLoopGroup("Selector", true, 1);
```

Comment:
Was this non-daemon thread still hanging around and that caused problems? It might be okay for this to be daemon threads (that allows the JVM to exit without shutting down the redis server) but we should probably still do some cleanup of these groups before we throw the ManagementException. We should try calling the "stop" method before throwing the ManagementException

Reply:
Good call, Darrel! Just updated it.  We decided to leave the selectorGroup as a daemon thread.